acir/src/circuit/directives.rs

```diff
@@ -192,21 +182,18 @@ impl Directive {
                 }
 
                 let radix = read_u32(&mut reader)?;
-
-                Ok(Directive::ToRadixLe { a, b, radix })
-            }
-            5 => {
-                let a = Expression::read(&mut reader)?;
-                let b_len = read_u32(&mut reader)?;
-                let mut b = Vec::with_capacity(b_len as usize);
-                for _ in 0..b_len {
-                    let witness = Witness(read_u32(&mut reader)?);
-                    b.push(witness)
+                let endianess = read_u32(&mut reader)?;
```

Comment:
not sure if I implemented write and read correctly sir. let me know if there's a better way